src/xGadgetron/pGadgetron/tests/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#========================================================================
+# Author: Edoardo Pasca
+# Copyright 2017 University College London
+# Copyright 2017 Science Technology Facilities Council
+#
+# This file is part of the CCP PETMR Synergistic Image Reconstruction Framework (SIRF) SuperBuild.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#=========================================================================
+include(FindPythonInterp)
```

Comment:
@KrisThielemans I'm assuming this is overriding travis' `PYTHON_EXECUTABLE`

Reply:
why would this happen only on one of the systems in the travis matrix?